modules/manage/pages/disaster-recovery/shadowing/setup.adoc

```diff
@@ -51,6 +51,64 @@ To set up Shadowing:
 * **Configure filters**: Define which topics, consumer groups, and ACLs should replicate by creating include/exclude patterns that match your disaster recovery requirements. See <<set-filters>>.
 * **Create a shadow link**: Establish the connection between clusters using `rpk`, the Admin API, or Redpanda Console with authentication and network settings. See <<create-a-shadow-link>>.
 
+== Shadow Link Tasks
+
+Shadow linking operates through three specialized tasks that handle different aspects of replication. Each task corresponds to a configuration section in your shadow link setup and runs continuously to maintain synchronization with the source cluster.
```

Comment:
Not a fan of including the number here ("three specialized tasks"). Inevitably, these things have a way of changing--maybe in another release there will be four, or five, or two, and you would have to catch & update this when that occurs. I'd delete the three. Your call though.